save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/projects/ProjectSettingsMenu.kt

```diff
@@ -82,18 +84,56 @@ private fun projectSettingsMenu() = FC<ProjectSettingsMenuProps> { props ->
     @Suppress("LOCAL_VARIABLE_EARLY_DECLARATION")
     val projectRef = useRef(props.project)
     val (draftProject, setDraftProject) = useState(props.project)
-
     useEffect(props.project) {
         if (projectRef.current !== props.project) {
             setDraftProject(props.project)
             projectRef.current = props.project
         }
     }
+    val navigate = useNavigate()
 
     val projectPath = props.project.let { "${it.organization.name}/${it.name}" }
+    val deleteProject = useDeferredRequest {
+        val responseFromDeleteProject = delete(
+            "$apiUrl/projects/$projectPath/delete",
+            jsonHeaders,
+            body = undefined,
+            loadingHandler = ::noopLoadingHandler,
+        )
+        if (responseFromDeleteProject.ok) {
+            navigate("/${FrontendRoutes.PROJECTS}")
+        }
+    }
 
-    val (wasConfirmationModalShown, showGlobalRoleWarning) = useGlobalRoleWarningCallback(props.updateNotificationMessage)
+    val updateProject = useDeferredRequest {
+        post(
+            "$apiUrl/projects/update",
+            jsonHeaders,
+            Json.encodeToString(draftProject.toDto()),
+            loadingHandler = ::loadingHandler,
+        ).let {
+            if (it.ok) {
+                props.onProjectUpdate(draftProject)
+            }
+        }
+    }
+
+    val (isModalOpen, setIsModalOpen) = useState(false)
+    displayModal(
```

Comment:
pls change on _displayConfirmationModal_

Reply:
![2022-09-29 15 04 52](https://user-images.githubusercontent.com/35039155/193026638-4489bd97-6897-4dd6-969d-ba9d3a94de1e.png)

Reply:
> I would not like to use it as far as I want to use custom buttons here

maybe extend the `confirmation` button in `displayConfirmationModal` with default value `Yes`

Reply:
Well, this is what we used to have. Let's better use `buttonBuilder`s and general `displayModal` instead of making simple methods more complicated.
Moreover, text is not the only thing that is needed to be changed. Style also should be changed.